app/actor/Renderer.scala

```diff
@@ -26,7 +26,7 @@ private[app] final class Renderer extends Actor {
     case streams: lila.streamer.LiveStreams.WithTitles => sender ! V.streamer.liveStreams(streams)
   }
 
-  private val spaceRegex = """\s{2,}""".r
+  private val spaceRegex = """\s{2,}+""".r
```

Comment:
I don't understand this change. `\s{2,}` already means "many spaces". What does `+` add?

Reply:
in this context the `+` makes the group posessive, disabling backtracking. not to be confused with the `+` that matches one or more times